src/handlers/fix-request-body.ts

```diff
@@ -3,6 +3,21 @@ import type * as express from 'express';
 import type { Request } from '../types';
 import * as querystring from 'querystring';
 
+/**
```

Comment:
Please move the `handlerFormDataBodyData` to the bottom of the file.

Please remove the `export` of this private function